install.sh

```diff
@@ -0,0 +1,17 @@
+if [ $# -eq 0 ]; then
+    echo "Please provide the name of the server to install to as a command line argument"
+    echo "For example: './install.sh mirebalais'"
+    exit 1
+fi
+
+# if there's a "config-pihemr" or "openmrs-config-pihemr" directory at the same level as this project,
+# run the install for it
+if [[ -f '../config-pihemr/install.sh' ]]; then
+    (cd ../config-pihemr && ./install.sh)
```

Comment:
Why not make this script independent of the script being in `config-pihemr`? Wrap the `mvn` command in a function like so:
```
mvn_compile () {
  mvn clean compile -DserverId=$1
}
```

And then you can do e.g.
```suggestion
    (cd ../config-pihemr && mvn_compile $1)
```

Then these scripts will be self-contained.

Reply:
There's a plus/minus to both approaches... it's self-contained, but now if you want to change the install instructions for the PIH EMR config you need to change it both in that install.sh and here... but I do agree, that reads cleaner, will take a look again tomorrow.

Reply:
Also could make sense just to remove the install.sh and watch.sh from PIH EMR because they aren't really needed (maybe that was part of what you were suggesting, actually)